aten/src/ATen/nn_parse.py

```diff
@@ -225,7 +225,7 @@ def unique_args(argslist):
     return result
 
 
-def function_info(name, arguments, cimpls, buffers, backends, inplace, scalar_check, backend_types):
+def function_info(name, arguments, cimpls, buffers, backends, inplace, scalar_check, backend_types, cpu_bf16):
```

Comment:
Is there any better way than adding additional parameter specific to one type only? How is Half doing it?

Reply:
Hi, @izdeby, I have packed the parser interface with one better way. Please review again